js/viz/sparklines/base_sparkline.js

```diff
@@ -272,12 +275,33 @@ BaseSparkline.prototype._setTooltipRendererOptions = function() {
     }
 };
 BaseSparkline.prototype._setTooltipOptions = function() {
-    const tooltip = this._tooltip;
-    const options = tooltip && this._getOption('tooltip');
-    tooltip && tooltip.update(_extend({}, options, {
-        customizeTooltip: generateCustomizeTooltipCallback(options.customizeTooltip, options.font, this.option('rtlEnabled')),
-        enabled: options.enabled && this._isTooltipEnabled()
-    }));
+    if(this._tooltip) {
+        const options = this._getOption('tooltip');
+        const defaultContentTemplate = this._getDefaultTooltipTemplate(options);
+        const contentTemplateOptions = defaultContentTemplate ? { contentTemplate: defaultContentTemplate } : {};
+        const optionsToUpdate = _extend(contentTemplateOptions, options, {
+            enabled: options.enabled && this._isTooltipEnabled()
+        });
+        this._tooltip.update(optionsToUpdate);
+    }
+};
+
+BaseSparkline.prototype._getDefaultTooltipTemplate = function(options) {
+    let defaultTemplateNeeded = false;
```

Comment:
The `defaultTemplateNeeded` can be declared with true, and than block with `else` can be removed